extensions/warp-ipfs/src/store/message.rs

```diff
@@ -1028,20 +1036,39 @@ impl MessageStore {
         invite_id: Uuid,
     ) -> Result<(), Error> {
         let inner = &*self.inner.read().await;
-        let community_meta = inner
-            .community_task
-            .get(&community_id)
-            .ok_or(Error::InvalidCommunity)?;
-        let (tx, rx) = oneshot::channel();
-        let _ = community_meta
-            .command_tx
-            .clone()
-            .send(CommunityTaskCommand::DeleteCommunityInvite {
-                invite_id,
-                response: tx,
-            })
-            .await;
-        rx.await.map_err(anyhow::Error::from)?
+        match inner.community_task.get(&community_id) {
+            None => {
+                let keypair = inner.root.keypair();
+
+                let event = CommunityJoinEvents::DeleteInvite { invite_id };
+                let bytes = serde_json::to_vec(&event)?;
```

Comment:
Do we need to convert this to a vector instead of just supplying `event`. to `PayloadBuilder::new`?

Reply:
removed. thanks